core/src/test/java/com/datastax/oss/driver/internal/core/metadata/schema/IndexMetadataTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.core.metadata.schema;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.metadata.schema.IndexKind;
+import com.datastax.oss.driver.api.core.metadata.schema.IndexMetadata;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class IndexMetadataTest {
+
+  private IndexMetadata indexMetadata;
+
+  @Before
+  public void before() {
+    final Map<String, String> options = new HashMap<>();
+    options.put("class_name", "com.datastax.MyClass");
```

Comment:
what is a reason for doing this in `@Before?` From what I see this is used in only one `@Test` so I guess it belongs to that test (given section)

Reply:
Good point. I started it in a `@Before` method as I thought I was going to have to mock things. If we decide to add more tests here, maybe there will be a more obvious reason for a `@Before` method, but for now I can just move it all inside the one test method.

Reply:
👍 to move it to the test method, it's a bit overkill right now.